### DISCUSSION TITLE
直播信息扩充组件新增“刷新”按钮，显示所有直播间

### DISCUSSION THREAD

**Discussion by 233YUZI** [the1812/Bilibili-Evolved#4035] [功能建议 / Ideas]
之前在用的一个油猴脚本“B站动态首页展示所有正在直播列表”https://greasyfork.org/scripts/438461 可以在列表上显示一个“刷新”按钮，这样就不需要单独刷新整个页面了，可惜那个作者不维护了。
![image](https://user-images.githubusercontent.com/69708373/222162561-64a4756f-99a8-495c-9857-c8c36686a610.png)
并且能否显示所有直播间呢？而不是只有10个，或者给个开关也行

**Comment by Geeyun-JY3** [comment]
`直播信息扩充` 这功能最近坏了：#3917、#4034

**Comment by the1812** [comment]
借这次坏了的机会, 我重新搞一个新的覆盖上去吧
